cmd/bsky-webhook/main.go

```diff
@@ -42,7 +43,10 @@ var (
 		"https://bsky.social"), "bluesky PDS server URL")
 	watchWord = flag.String("watch-word", envOr("WATCH_WORD", "tailscale"),
 		"the word to watch out for. may be multiple words in future (required)")
-
+	watchWords = flag.String("watch-words", envOr("WATCH_WORDS", ""), // "word1;word2"
+		"the words to watch out for. if watch-word is also specified, they are added")
```

Comment:
Separately: I guess you're adding a new flag rather than extending the existing one to avoid breaking existing use. Should we maybe make them mutually exclusive to make a clean break?

Reply:
Yes, and good point